modules/edu.gemini.web.client.facades/src/main/scala/edu/gemini/web/client/style/package.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package edu.gemini.web.client
+
+import japgolly.scalajs.react.vdom.html_<^._
+import cats.{Eq, Monoid}
+import cats.implicits._
+
+package style {
+  final case class GStyle(htmlClass: String)
+
+  object GStyle {
+    val Zero: GStyle = GStyle("")
+
+    implicit val eq: Eq[GStyle] = Eq.by(_.htmlClass)
+
+    implicit val monoid: Monoid[GStyle] = new Monoid[GStyle] {
+      override def empty = Zero
+      def combine(x: GStyle, y: GStyle): GStyle = (x, y) match {
+        case (x, y) if x === Zero && y === Zero => Zero
+        case (x, y) if x =!= Zero && y === Zero => x
+        case (x, y) if x === Zero && y =!= Zero => y
+        case (x, y)                             => GStyle(s"$x $y")
```

Comment:
You don't have a `toString` defined … are you sure you don't want `s"${x.htmlClass} ${y.htmlClass)"` here?

This is kind of complicated … Is there a reason this isn't encoded as `GStyle(styles: List[String])`? If you do it that way you can get a monoid via `Monoid[List[String]].imap(...)(...)` and can get `htmlClass` via `styles.intercalate(" ")`.